includes/theme-support/class-llms-twenty-twenty-two.php

```diff
@@ -92,7 +92,7 @@ protected static function generate_inline_styles( $context = null ) {
 			$styles[] = '.llms-form-field input, .llms-form-field textarea, .llms-form-field select { padding: 6px 10px }';
 
 			// Question layout.
-			$styles[] = '.llms-question-wrapper ol.llms-question-choices li.llms-choice .llms-choice-text { width: calc( 100% - 110px); }';
+			$styles[] = '.llms-question-wrapper ol.llms-question-choices li.llms-choice .llms-choice-text { width: calc( 100% - 110px); margin-top: 0; }';
```

Comment:
I tested the margin thing as well and find it was needed... Without it I had the label horizontally misaligned (compared to the A|B...)
Will give it another shot.
I'm now realizing this method requires a changelog and the file's `@version` would need to be changed.
I don't remember the reason for that `calc` :(

Reply:
No worries we can just leave it alone. I thought it might not be needed but if you can't remember it's not hurting anyone so I say let's leave it alone.

Reply:
I looked into this:
- by default the label is displayed as `inline`, with no width specified.
- when so, its`.llms-choice-text` children with the width set as [width: calc(100% - 60px);](https://github.com/gocodebox/lifterlms/blob/dcc1eab319394757d69f12eade07249fa47166d3/_private/scss/frontend/_llms-quizzes.scss#L322) dropped on a new line. So I found that increasing that 60px to 110px was fine to have it back on the same line of the marker. Actually a lower value (95px) was good enough.
- Turning the label display into `block`, makes the rule `width: calc( 100% - 110px);` useless, but the` margin-top:0`; is needed, agreed? Is this what you meant with 
> renders this particular rule unnecessary

?
If so I agree. <3